redfish-core/lib/systems.hpp

```diff
@@ -2076,6 +2076,88 @@ inline void getEnabledPanelFunctions(
         });
 }
 
+/**
+ * Execute a Panel Enabled Function
+ */
+inline void
+    executePanelFunction(const std::shared_ptr<bmcweb::AsyncResp>& asyncResp,
+                         const uint8_t funcNo)
+{
+    BMCWEB_LOG_DEBUG << "Execute Panel function";
```

Comment:
It might be helpful to include the funcNo in the DEBUG output.

Reply:
Done